### PR TITLE
Fix/refactor global state

### PR DESCRIPTION
**Pull request by Diegogagan2587**
# Implemented Changes:

- [ ] Refactor all local and global states to improve navigation time within the app
- [ ] Fix My reservations not loading properly because the pending request
- [ ] Adding or removing concerts will update the Concerts List
- [ ] Reserve will update reservations list
- [ ] Additional Fixes
    - [ ] Fix the Navigation bar when the user is in My Reservations: Now the user can scroll and the navigation bar will remain the same
    - [ ] Fix overflow within reservation cards when the text is too long.

Before modification:
image

After modification:
image

image

image

### CODE REVIEW

File: src/pages/MyReservationsPage.jsx
Comment (rudicarrilloypr) [inline src/pages/MyReservationsPage.jsx:1]: the new reservation I'm creating is not showing on my reservations, kindly fix that.

File: src/redux/slices/reservationSlice.js
Comment (rudicarrilloypr) [inline src/redux/slices/reservationSlice.js:1]: I'm trying to create a new reservation and it shows multiple reservations that I haven't created, kindly fix that.

**Review by rudicarrilloypr — CHANGES_REQUESTED** [review]
> Hi paisa @Diegogagan2587    ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next task but you are almost there!

 almost
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used Gitflow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me (@rudicarrilloypr) in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danifromecuador — APPROVED** [review]
Hi @Diegogagan2587

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

## Highlights
 - All project requirements have been implemented ✅

chuck-norris-thumbs-up

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @danifromecuador .

**Pull request merged**